website/src/categories/whats-new.md

```diff
@@ -21,6 +21,39 @@ eleventyNavigation:
 
 <section>
 
+## Enhancements and Insights: New Components, Bug Fixes, Styling, and Workshops
+<small>June 2024</small>
+
+### New components
+- [button bar](/categories/components/button-bar/usage/) (stable)
```

Comment:
Maybe we can use `<ds-status status="stable"></ds-status>` here and for other `<ds-status status="draft"></ds-status>` and `<ds-status status="preview"></ds-status>` instead of writing status only in the brackets?:)